ProjectManagmentSystem/src/Modules/ProjectsModule/components/ProjectsList/ProjectsList.tsx

```diff
@@ -1,7 +1,307 @@
+                      <div className="col-md-2">{project.task?.length}</div>
+                      <div className="col-md-2">
+                        {project.creationDate.slice(0, 10)}
+                      </div>
+                      <div className="col-md-2 bg-body-secondar bg-info-subtl w-fit p-md-0 px-md-3 px-lg- rounded-3 d-flex justify-content-center align-items-center">
+                        <div className="dropdown">
+                          <button
+                            className="btn"
+                            type="button"
+                            data-bs-toggle="dropdown"
+                            aria-expanded="false"
+                          >
+                            <i className="fa fa-ellipsis-vertical"></i>
+                          </button>
+                          <ul className="dropdown-menu">
+                            <li>
+                              <a className="dropdown-item" href="#">
+                                <i className="fa fa-eye text-info mx-2"></i>
+                                View
+                              </a>
+                            </li>
+                            <li>
+                              <a className="dropdown-item" href="#">
+                                <i className="fa fa-edit text-warning mx-2"></i>
+                                Edit
+                                {/* TODO:implement Update */}
+                              </a>
+                            </li>
+                            <li>
+                              <a
+                                onClick={() => handleDeleteShow(project.id)}
+                                className="dropdown-item"
+                                href="#"
+                              >
+                                <i className="fa fa-trash text-danger mx-2"></i>
+                                Delete
+                              </a>
+                            </li>
+                          </ul>
+                        </div>
+                      </div>
+                    </div>
+                  </li>
+                ))
+              ) : (
+                <li className="list-group-item">
+                  <NoData />
+                </li>
+              )}
+                      
+            </ul>
+            {/* TODO:implement Pagination */} <div className={`${Styles.pagination} text-muted`}>
+          <span>Showing</span>
+          <div className={Styles.contentSize}>
+            <div
+              className={Styles.pageSize}
+              onClick={() => setShowListSize(!showListSize)}
+            >
+              <span>{pageSize}</span>
+              <span>
+                <i className="fa-solid fa-chevron-down"></i>
+              </span>
+            </div>
+            {showListSize && (
+              <div className={`${Styles.listsPage} text-muted`}>
+                {totalNumberOfRecords.map((n) => (
+                  <span
```

Comment:
This should be a button, as it is intended to an interactive element that users need to click.

Reply:
Done in TasksList will apply in ProjectsList